### PR TITLE
Data sorting

### PR DESCRIPTION
**Pull request by sgiaccio**

### CODE REVIEW

File: server/surveyRdb/dbActions/tableViewQuery.js [hunk 1]
```diff
@@ -68,12 +69,12 @@ const queryRootTableByRecordKeys = async (survey, recordUuid, client) => {
     keyNodes.map(
       async node => {
         const colValue = await getColValue(node)
-        return `${getColName(node)} ${colValue === null ? ' IS NULL' : " = '" + colValue + "'"}`
+        return `${getColName(node)} ${colValue === null ? ' IS NULL' : ' = \'' + colValue + '\''}`
```
Comment (minotogna) [inline server/surveyRdb/dbActions/tableViewQuery.js]: is this escaping needed ?

File: webapp/commonComponents/form/input.js [hunk 2]
```diff
@@ -24,21 +24,24 @@ export const Input = React.forwardRef((props, ref) => {
   } = props
 
   return (
-    <TooltipError messages={validation.errors}>
+    <Tooltip
+      messages={validation.errors}
+      type={`${validation.errors ? 'error' : ''}`}
+      style={{ display: 'grid', width: '100%', height: '100%' }}>
```
Comment (minotogna) [inline webapp/commonComponents/form/input.js]: why would you need this inline style?

File: webapp/commonComponents/tooltip.js [hunk 3]
```diff
@@ -3,42 +3,53 @@ import './tooltip.scss'
...
+    </div>
+  }
 }
 
-export const TooltipError = (props) => (
-  <Tooltip {...props}
-           type="error"/>
-)
+Tooltip.defaultProps = {
+  message: null,
```
Comment (minotogna) [inline webapp/commonComponents/tooltip.js]: I think it's messages... could you also add the other props?

File: server/surveyRdb/dbActions/tableViewQuery.js (hunk 1 above)
Comment (sgiaccio) [reply server/surveyRdb/dbActions/tableViewQuery.js]: It's not... Must have been the editor...

File: webapp/commonComponents/form/input.js (hunk 2 above)
Comment (sgiaccio) [reply webapp/commonComponents/form/input.js]: Yes it doesn't look good, I'll find a better solution

File: webapp/commonComponents/tooltip.js (hunk 3 above)
Comment (sgiaccio) [reply webapp/commonComponents/tooltip.js]: Will do

**Comment by minotogna** [comment]
Issues:
When I set the filter, then i navigate to another page, when i come back to the dataQuery on mouse over the filter button the sort is not displayed

**Pull request merged**